Makefile

```diff
@@ -207,3 +225,36 @@ build-integration-cli-on-swarm: build ## build images and binary for running int
 	docker exec $(tmp) go build -buildmode=pie -o /worker github.com/docker/docker/hack/integration-cli-on-swarm/agent/worker
 	docker commit -c 'ENTRYPOINT ["/worker"]' $(tmp) $(INTEGRATION_CLI_WORKER_IMAGE)
 	docker rm -f $(tmp)
+
+# TODO: Eventually clean this up when we release an image with a manifest
+DOCKER2OCI=bundles/docker2oci
+$(DOCKER2OCI):
+	-chown -R $(shell id -u):$(shell id -g) $(@D)
+	docker run --name docker2oci $(GO_IMAGE) sh -c 'go get github.com/coolljt0725/docker2oci'
+	mkdir -p $(@D)
+	docker cp docker2oci:/go/bin/docker2oci "$@"
+	docker rm -f docker2oci
+	chown -R $(shell id -u):$(shell id -g) $(@D)
+
+clean-image:
+	-rm $(DOCKER_HUB_ORG)_$(ENGINE_IMAGE)_$(IMAGE_VERSION).tar
+	-docker rmi $(DOCKER_HUB_ORG)/$(ENGINE_IMAGE):$(IMAGE_VERSION)
+
+image: $(DOCKER_HUB_ORG)_$(ENGINE_IMAGE)_$(IMAGE_VERSION).tar
+	@echo ""
+	@echo "To test your local build:"
+	@echo "ctr --namespace docker image import --oci-name docker.io/moby/engine $<"
```

Comment:
FYI: This is going to be changed: https://github.com/containerd/containerd/pull/2633

cc @dmcgowan